packages/web-app-files/src/components/SideBar/Details/FileDetails.vue

```diff
@@ -131,16 +198,85 @@ export default defineComponent({
     sharedByDisplayName: '',
     sharedTime: 0,
     sharedItem: null,
-    shareIndicators: []
+    shareIndicators: [],
+    copiedDirect: false,
+    copiedEos: false,
+    timeout: null
   }),
   computed: {
-    ...mapGetters('Files', ['versions', 'sharesTree', 'sharesTreeLoading']),
+    ...mapGetters('Files', [
+      'versions',
+      'sharesTree',
+      'sharesTreeLoading',
+      'currentFileOutgoingCollaborators'
+    ]),
     ...mapGetters(['user', 'getToken', 'configuration']),
 
+    sharedWithLabel() {
+      return this.$gettext('Shared with')
+    },
     file() {
       return this.displayedItem.value
     },
 
+    isEos() {
+      return !!this.configuration.options?.eos
```

Comment:
Please add the eos config options to https://owncloud.dev/clients/web/getting-started/#options (markdown file in the `docs/` directory in this repo) and ideally also with a default value in https://github.com/owncloud/web/blob/72fba4481b9d3b9592ed064b0f39d8503518df66/packages/web-runtime/src/store/config.js#L40